src/app/js/src/SampleApp.jsx

```diff
@@ -244,9 +243,24 @@ class App extends Component {
         );
         let sideBar;
         if (this.props.isCollapsed) {
+            const styleWhiteText = {
+                color: '#fff',
+            };
+            const styleBlueBackground = {
+                backgroundColor: '#00bcd6',
+            };
```

Comment:
Our team typically hasn't implemented inline style rules in react components, even though it's something that is supported and sometimes recommended. Consider using classes for these instead.

Reply:
Material UI expects a prop here and the same in a class didn't work for me

Reply:
Got it. thanks.